src/windows/wslaservice/exe/WSLAVirtualMachine.cpp

```diff
@@ -209,14 +200,14 @@ void WSLAVirtualMachine::Start()
     kernelCmdLine += L" hv_utils.timesync_implicit=1";
 
     wil::unique_handle dmesgOutput;
-    if (m_settings.DmesgOutput != 0)
+    if (m_settings.DmesgHandle)
     {
-        dmesgOutput.reset(wsl::windows::common::wslutil::DuplicateHandleFromCallingProcess(ULongToHandle(m_settings.DmesgOutput)));
+        dmesgOutput = std::move(m_settings.DmesgHandle);
```

Comment:
can this whole condition be removed and just always move the dmesg handle? won't harm anythigng if it's empty.